### PR TITLE
Fixes, optimizations and new features

### PR DESCRIPTION
**Pull request by zTrap**
# Views module: 
### public changes
- Added new methods for `IconicsTextView` Family
   - `void:setCheckedDrawableForAll(Drawable)`
   - `void:setDrawableForAll(Drawable)`
### internal changes
- Removed unused internal code in `IconicsMenuInflaterUtil`
- Removed wrong getting from attributes
- Fixed calling wrong method in `IconicsBundle`
- Added constant fields in `IconicsView`
- Separated method `void:readIconicsImageView(TypedArray, IconBundle)` was moved back into `IconicsViewsAttrsReader`
- Deleted `IconicsCoreAttrsReader`
- Added support nullable `IconBundle`
- Fixed usage wrong attributes for `IconicsCheckableTextView`

# Core module:
### public changes
 - `IconicsDravable.ANDROID_ACTIONBAR_ICON_SIZE_DP` and `IconicsDravable.ANDROID_ACTIONBAR_ICON_SIZE_PADDING_DP` now is ***deprecated***. 
Use `IconicsDravable.TOOLBAR_ICON_SIZE` and `IconicsDravable.TOOLBAR_ICON_PADDING` instead
### internal changes
- Removed unused `strings.xml`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -264,10 +264,12 @@ new Iconics.IconicsBuilder().ctx(this)
 ### String icon-key or typeface enum
 Sometimes you won't like to use the icon-key ("faw-adjust") like this, but use the enum provided by a specific font. Both is valid:
 ```java
-  new IconicsDrawable(this, "faw-adjust").actionBarSize()
+  new IconicsDrawable(this, "faw-adjust").actionBar()
 ```
 ```java
-  new IconicsDrawable(this, FontAwesome.Icon.faw_adjust).sizeDp(24)
+  new IconicsDrawable(this, FontAwesome.Icon.faw_adjust)
```
Comment (mikepenz) [inline README.md]: well I would replace them with some numbers, much shorter than the internal static fields :D

Comment (zTrap) [inline README.md]: so, also I would rename fields to more shorter :) 

Like that: 
ANDROID_ACTIONBAR_ICON_SIZE_DP -> ACTIONBAR_ICON_SIZE_DP
ANDROID_ACTIONBAR_ICON_SIZE_PADDING_DP -> ACTIONBAR_ICON_SIZE_PADDING_DP
It's little bit shorter :)

Comment (mikepenz) [inline README.md]: TOOLBAR_ICON_SIZE
TOOLBAR_ICON_PADDING

Comment (zTrap) [inline README.md]: _DP is important I think

Comment (zTrap) [inline README.md]: But, if you want, I can use name w/o *_DP*, place some javadoc for this fields, and write in docs about used dimensions

**Comment by mikepenz** [comment]
@zTrap sorry I was really busy the last few days. I'll take a look in the upcoming days. thanks

**Pull request merged**